docs/Readme.md

```diff
@@ -0,0 +1,31 @@
+GlueStick
+=========
+
```

Comment:
This is going to be a new refined Readme customer oriented. I propose we keep the project Readme as dev-oriented as possible with minimal content (requirements etc.) 

Selling points can go here.